packages/react-native/refresh-example.sh

```diff
@@ -1,28 +1,22 @@
 #!/bin/bash
 set -e
 
-echo "You should run this from the packages/react-native directory inside the cloned invertase/notifee repo"
-echo "You should only do this when your git working set is completely clean (e.g., git reset --hard)"
-echo "You must have already run \`yarn\` in the repository so \`npx react-native\` will work"
-echo "This scaffolding refresh has been tested on macOS, not on windows or linux"
+# echo "You should only do this when your git working set is completely clean (e.g., git reset --hard)"
+# echo "You must have already run \`yarn\` in the repository so \`npx react-native\` will work"
+# echo "This scaffolding refresh has been tested on macOS, not on windows or linux"
 
 # Copy the important files out temporarily
 if [ -d TEMP ]; then
   echo "TEMP directory already exists - we use that to store files while refreshing."
-#   exit 1
 elif [ -d example ]; then
   echo "Saving files to TEMP while refreshing scaffolding..."
   mkdir -p TEMP/android
   mkdir -p TEMP/android/app/src/main/java/com/example
   cp example/README.md TEMP/
   cp example/android/local.properties TEMP/android/ || true
-  cp example/android/app/src/main/AndroidManifest.xml TEMP/android/app/src/main/
-  cp example/android/app/src/main/java/com/example/MainActivity.java TEMP/android/app/src/main/java/com/example/
-  cp example/android/app/src/main/java/com/example/CustomActivity.java TEMP/android/app/src/main/java/com/example/
```

Comment:
There was some customization in those I think - if not I would not have preserved them :thinking:

Reply:
yeah i was going to add them later as they are for fullScreenActivity but i've adding some `sed` commands to handle the customisations, should be more future proof 😎